### DISCUSSION TITLE
Automatic services

### DISCUSSION THREAD

**Discussion by f4emm** [foldynl/QLog#135] [Ideas]
Hello,
Would it be possible to have an automatic service dispatch function?
Thanks.
Have a good day.
F4EMM

**Comment by foldynl** [comment]
things regarding external services are quite problematic. Each service handles log reception differently. Some are not able to work in online mode.  In order to at least unify the access to the services, I have not implemented the online upload function. 

I am not planning to do so at this moment. But this doesn't mean that it won't be there someday later.

QLog is an open source project. Unlike commercial productions, the long-term plan is only to make QLog usable for day-to-day operation. Specific steps how to achieve it are not predetermined.

However, I am not against it when someone implements this functionality and sends it as PR (patch)

**Reply by f4emm** [reply]
Hello,
Thank you for your reply.
Have a good day.
